org.endlessos.Key.yaml

```
@@ -0,0 +1,81 @@
+app-id: org.endlessos.Key
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: /app/bin/kolibri-gnome
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --system-talk-name=org.endlessos.Key.Daemon
+  - --env=KOLIBRI_HOME=~/.var/app/org.endlessos.Key/data/kolibri
+  - --env=KOLIBRI_HTTP_PORT=0
+  - --env=PYTHONPATH=/app/kolibri-plugins/lib/python
+
+add-extensions:
+  org.learningequality.Kolibri.Content:
+    version: '1.0'
+    directory: share/kolibri-content
+    subdirectories: true
+    no-autodownload: true
+
+  org.learningequality.Kolibri.Plugin:
+    version: '1.0'
+    directory: kolibri-plugins
+    merge-dirs: lib/python
+    subdirectories: true
+    no-autodownload: true
+
+build-options:
+  env:
+    KOLIBRI_MODULE_PATH: /app/lib/python3.10/site-packages/kolibri
+  test-args:
+    - --socket=fallback-x11
+    - --socket=wayland
+
+modules:
+  - modules/iproute2.json
+  - modules/python3-markdown.json
+  - modules/python3-kolibri.json
+  - modules/python3-kolibri-pytz.json
+  - modules/python3-kolibri-app-desktop-xdg-plugin.json
+  - modules/python3-kolibri-desktop-auth-plugin.json
+  - modules/kolibri-home-template.json
+  - modules/python3-setproctitle.json
+  - modules/python3-virtualenv-api.json
+  - modules/kolibri-content-dir.json
+  - modules/kolibri-plugins-dir.json
+
+  - name: kolibri-gnome
+    buildsystem: meson
+    builddir: true
+    run-tests: true
+    config-opts:
+      - -Dkolibri_home_template_dir=/app/share/kolibri-home-template
+      - -Dprofile=default
+    sources:
+      - type: git
+        url: https://github.com/endlessm/endless-key-flatpak.git
+        tag: endlesskey_1.0.0
+        commit: 927b8022cc25facc11fe38ccd20ab2c04b49da1b
```

I changed the versioning and tagging a bit, so I think this should be:

```suggestion
        tag: v0.1
        commit: a9781eee6379ae2196ca2c81a65ee20a0a37aafe
```